tests/stub-cards/card-definition-id-does-not-match-package-name/node_modules/person-card/schema.js

```diff
@@ -0,0 +1,21 @@
+const JSONAPIFactory = require('@cardstack/test-support/jsonapi-factory');
+module.exports = function() {
+  let factory = new JSONAPIFactory();
+  // Person Card Schema
+  factory.addResource('card-definitions', 'not-person-card')
+    .withRelated('model', factory.addResource('content-types', 'people')
+      .withRelated('fields', [
+        factory.addResource('fields').withAttributes({
+          name: 'name',
```

Comment:
Now I'm even more confused about `member-name`, because we already have `name`.

Reply:
this was me not being complete in switching to `member-name`